nuxeo-core/nuxeo-core-cache/src/main/java/org/nuxeo/ecm/core/transientstore/api/TransientStoreService.java

```diff
@@ -41,4 +41,11 @@
      * Triggers Garbage collecting of all {@link TransientStore}
      */
     void doGC();
+
+    /**
+     * @since 10.1
+     * @param name Name of the transient store to check existence
+     * @return true or false
+     */
+    boolean containsStore(String name);
```

Comment:
It was used to check for Transient Store existance. But either way, i think this method should stay, since `getStore()` returns the default if the requested one doesn't exist.

Reply:
Yes, that's what is stated in the Javadoc of `getStore()`.
Not sure I see any use case but why not.

Reply:
There is no way of knowing if a certain `TransientStore` exists. For example, i used to have a validation to check for the existence of the `TransientStore`